smokeTest.js

```diff
@@ -87,23 +117,49 @@ async function im21Door(coreID, signal) {
 async function smokeTest(recipientPhoneNumber, twilioPhoneNumber) {
   helpers.log('Smoke testing - please do not terminate this script before it finishes running')
   try {
-    await setup(recipientPhoneNumber, twilioPhoneNumber, testLocation1Id, door_coreID, radar_coreID)
+    helpers.log('Running XeThru Smoke Tests')
```

Comment:
Could you please update the README to include what we expect when we run the smoke tests?